docs/technical-report-1.md

```diff
@@ -247,317 +247,317 @@ Limiting nodes to a maximum of one vote would likely be safe if the mean number
+|     350 B |                      214 |                     403 |                                   80.6 % |                                      19.4% |  
+|     400 B |                      187 |                     421 |                                   84.2 % |                                      15.8% |  
+|     450 B |                      166 |                     439 |                                   87.8 % |                                      12.2% |  
+|     500 B |                      150 |                     457 |                                   91.4 % |                                       8.6% |  
+  
+Using the previously mentioned ALBA parameters and setting $n_p / n = 0.9$, we show below measurements for the unoptimized "centralized telescope" version of the pipeline. This uses Blake2s hashes, but other algorithms such as SHA256, Keccak256, and Poseidon are possible alternatives. On the particular machine used for these benchmarks, proof took approximately 9.0 ms and verification took 85 μs; note that the ~148 votes in the certificate would each have to be verified, too. The optimized version of ALBA, which packs data more efficiently, might result in certificates that are 40% smaller than the nominal 75 kB of the certificate in this example.  
+  
+![Proof and verification times for ALBA certificates relevant to Leios](../images/leios-alba.svg)  
+  
+  
+#### BLS certificate  
+  
+The BLS certificate describe in the Leios paper does not compress votes the way that ALBA does, so votes must be fully stored in the certificate. However, only a quorum of votes, not all of them, need to be stored, which affords a type of compression. Additionally, two aggregate signature must also be stored. A certificate for a quorum of votes attested by ephemeral keys would fit in a Praos block, but a certificate attested by KES keys is about 50% too large.  
+  
+| Method        | Common to all votes | Specific to individual vote | Aggregation | Number of votes | Certificate of all votes | Quorum | Certificate of quorum |  
+| ------------- | ------------------: | --------------------------: | ----------: | --------------: | -----------------------: | -----: | --------------------: |  
+| Ephemeral key |                32 B |                       209 B |       192 B |             500 |                   105 kB |    60% |                 63 kB |  
+| KES key       |                64 B |                       529 B |       192 B |             500 |                   265 kB |    60% |                159 kB |  
+  
+Also recall that any ephemeral keys would have to be registered on-chain, consuming additional precious space and complicating bookkeeping: that would only provide benefit if the keys were reused for many votes. It might also be possible to only store the Merkle proof for the KES key only at the start of the KES period (e.g., every 36 hours).  
```

Comment:
The problem with ephemeral keys  is not so much the space that it would require on-chain, this can be as low as 32 bytes, but the extra work needed (we would need to make sure that everyone stores these keys and that a commitment of them is published on-chain) as you actually described for the Merkle proof of KES keys.

Reply:
Clarified in 3918063c5755ea0b305fdf19035309fb228d1a25.